docs-2.0/4.deployment-and-installation/5.zone.md

```diff
@@ -107,40 +119,19 @@ nebula> DESC ZONE az1
 | Hosts           | Port |
 +-----------------+------+
 | "192.168.8.111" | 7779 |
-| "192.168.8.111" | 9779 |
+| "192.168.8.112" | 9779 |
 +-----------------+------+
 ```
 
-### Create a space in the specified Zone
-
-```ngql
-CREATE SPACE IF NOT EXISTS  (
-    [partition_num = <partition_number>,]
-    [replica_factor = <replica_number>,]
-    vid_type = {FIXED_STRING(<N>) | INT[64]}
-    )
-    [COMMENT = '<comment>'] 
-    [ON <zone_list>];
-```
-
-!!! note
+### Create a space in the specified Zones
 
-    - The Zone specified when creating a graph space must be one or more of the values in the `--zone_list` of the Meta configuration file, otherwise the graph space cannot be created.
-    - The Zone specified when creating a graph space must contain at least one Storage node, otherwise the graph space cannot be created.
-    - The number of partition replicas specified when creating a graph space must be less than or equal to the number of Zones, otherwise, the graph space cannot be created.
+The syntax for creating a graph space within a Zone is the same as in [Creating Graph Space](../3.ngql-guide/9.space-statements/1.create-space.md). 
 
+However, during graph space creation, the system automatically recognizes the `--zone_list` value from the Meta configuration file. If this value is not empty and the number of Zones matches the partition replica count specified by `replica_factor`, the graph space's replicas will be evenly distributed across the Zones in `--zone_list`. If the specified replica count doesn't match the number of Zones, graph space creation will fail.
```

Comment:
这里的value是指 [ON <zone_list>]里面的zone_list吧？如果是，请看下面修改。

`If this value is not empty and it is the same as the value specified in \`--zone_list\` from the Meta configuration file .... Otherwise, graph space creation will fail.